src/main/java/com/microsoft/jenkins/appservice/WebAppDeploymentCommandContext.java

```diff
@@ -147,7 +153,7 @@ public void configure(
             }
         } else if (WebAppUtils.isJavaApp(app)) {
             // For Java application, use WAR deployment
-            builder.withStartCommand(WarDeployCommand.class);
+            builder.withStartCommand(FileDeployCommand.class);
```

Comment:
Personally speaking, I would rather write two commands `WarDeployCommand` and `ZipDeployCommand` and set different start command here according to deploy type. This makes code cleaner and easier to test. But that's up to you. Just a suggestion.

Reply:
Make sense. But won't change since the ZIP&WAR deploy commands are not complicate and they are only for Java apps.